python/nammu/controller/AtfAreaController.py

```diff
@@ -130,10 +130,19 @@ def redo(self):
 
     def __getattr__(self, name):
         '''
-        Calls to copy, paste and cut methods are just passed to text area.
+        Calls to copy, paste and cut, getSelectedText, getSelectionStart
+        methods are just passed to text area. replaceSelection and
+        setCaretPosition require the wrapper to handle their args.
         '''
-        if name in ('copy', 'paste', 'cut'):
-            return getattr(self.view.edit_area, name)
+        if name in ('copy', 'paste', 'cut', 'getSelectedText',
+                    'getSelectionStart'):
+            return getattr(self.edit_area, name)
+
+        elif name in ('replaceSelection', 'setCaretPosition'):
+
+            def wrapper(*args, **kw):
+                return getattr(self.edit_area, name)(*args, **kw)
+            return wrapper
```

Comment:
Yeah, there is I have not found any change in behaviour of any of the functions we have moved into `getattr`.